tests/antool/decompress-bin-arch.sh

```diff
@@ -59,13 +59,26 @@ if [ "$FILES_ARCH" == "" ]; then
 fi
 
 echo -n "Decompressing vltrace's binary logs... "
+
 for file in $FILES_ARCH; do
 	bzip2 -d -f -k $file
 done
+
+TEMP_FILE=$(mktemp)
+
+# concatenate log files with syscall table
+for file in $SRC_DIR/$MASK_BIN; do
+	cp $file $TEMP_FILE
+	cp $SRC_DIR/$SYSCALL_TABLE $file
+	cat $TEMP_FILE >> $file
+done
```

Comment:
New commit message:

antool: check signature, version and architecture of vltrace log

antool.py does not need separate input file 'syscall_table.dat',
because the syscall table is a part of vltrace binary log now,
so the '-t' option was removed.

In order to keep compressed vltrace binary logs as small as possible,
the syscall table is cut off of each binary log during 'make regenerate'
and kept as a single file in the repo. It is concatenated with the rest
of log during decompression of logs.

These two changes are merged into one commit in order to update
the compressed vltrace binary logs only once.